Content.Server/Animals/Components/PettableComponent.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Content.Shared.Sound;
+using Content.Server.Animals.Systems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Animals.Components
+{
+    [RegisterComponent, Friend(typeof(PettingSystem))]
+    public class PettableComponent : Component
```

Comment:
```suggestion
    public sealed class PettableComponent : Component
```

Will need renaming too.